Automattic/woocommerce-payments
# Remove unused file

Fixes #6304

#### Changes proposed in this Pull Request









This PR removes the unused `includes/admin/class-wc-rest-user-exists-controller.php` file mentioned on #6304 as the references to it were removed 10 months ago, so the problem that happened when updating WCPay should not be a problem anymore.

I couldn't find a way to prevent versions before 5.8.1 to run the `init_rest_api` while updating.

#### Testing instructions





1. Install WCPay 5.9.0 or newer.
2. Go through onboarding.
3. Build the plugin locally.
4. Upgrade the site to the plugin built in step 3.
5. No fatal error should be thrown. 

-------------------


**Post merge**
